thriftpy/parser/parser.py

```diff
@@ -399,6 +400,8 @@ def p_simple_base_type(p):  # noqa
         p[0] = TType.BOOL
     if p[1] == 'byte':
         p[0] = TType.BYTE
+    if p[1] == 'i8':
+        p[0] = TType.BYTE
```

Comment:
It's not good to reuse `TType.BYTE`, as `I8` you defined in `p_simple_base_type ` is useless now.